### PR TITLE
performance tests implementation

### PR DESCRIPTION
**Pull request by ChGen**
Here's implementation of performance testing for Artipie. Part of  https://github.com/artipie/benchmarks/issues/64
It runs with self-hosted GitHub runners. Currently tests run for localhost and by ethernet between two machines. Localhost results are more stable, due to escaping all extra routing and security software checks.
This workflows do tests and generate two sets of graphs: for git tags: `v*` and `t*` (release tags and test tags)

### CODE REVIEW

**Comment by ChGen** [comment]
@dgarus , please review

File: .github/workflows/perftest-server.yml [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        run: test ! -f $HOME/.benchvars && exit 1;exit 0
+      - name: Prepare docker env
+        run: |
+          docker info
+          docker stop artipie || :
+          docker rm -fv artipie || :
+          docker image rm -f artipie/artipie:1.0-SNAPSHOT || :
+          docker ps
+      - name: Maven build
+        run: timeout 600 mvn clean install
```
Comment (olenagerasimova) [inline .github/workflows/perftest-server.yml:25]: I'd suggest skip tests and use docker profile
`mvn clean install -Pdocker -DskipTests`

**Review by olenagerasimova — COMMENTED** [review]
@ChGen thanks, just one suggestion

File: .github/workflows/perftest-server.yml (hunk 1 above)
Comment (ChGen) [reply .github/workflows/perftest-server.yml:25]: @olenagerasimova , thanks.

**Review by olenagerasimova — APPROVED** [review]
looks good to me

**Review by dgarus — APPROVED** [review]
lgtm

**Pull request merged**